src/pages/home/sidebar/OptionRow.js

```diff
@@ -33,7 +33,7 @@ const propTypes = {
     optionIsFocused: PropTypes.bool.isRequired,
 
     // A function that is called when an option is selected. Selected option is passed as a param
-    onSelectRow: PropTypes.func.isRequired,
+    onSelectRow: PropTypes.func,
```

Comment:
Why was this change necessary?

Reply:
This was necessary because I don't use onSelectRow  in my display component and the PropTypes was logging errors. I'm not sure whether it'd break other components though because we are setting a required option to be optional. The other way around would definitely break it. I thought this should be okay.